buidler-app/contracts/Payroll.sol

```diff
@@ -90,6 +90,7 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         uint256 indexed employeeId,
         address indexed accountAddress,
         address indexed token,
+        uint256 denominationAllocation,
```

Comment:
There's likely some tests you'll have to update if you're adding this here. Since the tests are now running in the buidler folder on `master` you can run them and check.